content/reference-data-api.md

```diff
@@ -3006,6 +3212,190 @@ Response:
 
 
 
+## Get Account Transaction Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
+
+<!--<div class='multicode'>-->
+
+*REST*
+
+```
+GET /v2/accounts/{:address}/stats/transactions
+```
+
+<!--</div>-->
+
+[Try it! >](data-api-v2-tool.html#get-account-transaction-stats)
+
+This method requires the following URL parameters:
+
+| Field    | Value  | Description |
+|----------|--------|-------------|
+| :address | String | Ripple address to query |
```

Comment:
*Minor and superficial:* It looks like all the existing open-source reference docs have "Fields" in regular font. For the Admin API, I used backticks around the field names because I thought the monospace font looked better in the PDF. Probably better to maintain consistency at this point than bother changing what's there already, but I'd be interested in your take for future docs, style guide, etc.